docs/python/settings-reference.md

```diff
@@ -66,6 +66,7 @@ The language server settings apply when `python.languageServer` is `Pylance` or
 | importFormat | `absolute`| Defines the default format when auto importing modules. Accepted values are `absolute` or `relative`. |
 | inlayHints.variableTypes | false | Whether to display inlay hints for variable types. Accepted values are `true` or `false`. |
 | inlayHints.functionReturnTypes | false |  Whether to display inlay hints for function return types.  Accepted values are `true` or `false`. |
+| inlayHints.pytestParameters | false | Whether to display inlay hints for pytest fixture argument types. Accepted values are `true` or `false`. |
```

Comment:
Why is this a separate setting instead of just being part of having inlay hints turned on/off for parameters?

Reply:
we don't offer type hints for parameters, only variable and function return types 😁